### PR TITLE
refactor: init now uses async await instead of promises

### PR DESCRIPTION
**Pull request by jeffladiray**
## Pull Request checklist:

- [x] Write an explicit title for the Pull Request, following [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [x] Test manually the implemented changes
- [ ] Review my own code (indentation, syntax, style, simplicity, readability)
- [ ] Wonder if you can improve the existing code

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -94,23 +94,97 @@ exports.ensureAuthenticated = (request, response, next) => {
...
+  await Schemas.perform(
+    Implementation,
+    configStore.integrator,
+    models,
+    lianaOptions,
+  );
+  return models;
+}
+
+function generateAndSendSchema(opts) {
```
Comment (jeffladiray) [inline src/index.js:111]: This was just move to a separated function - no code changes here

File: src/index.js [hunk 2]
```diff
@@ -198,26 +272,25 @@ exports.init = (Implementation) => {
...
+    } else {
+      directorySmartImplementation = `${path.resolve('.')}/forest`;
+    }
+
+    if (fs.existsSync(directorySmartImplementation)) {
+      await requireAllModels(directorySmartImplementation);
+    }
+    if (opts.configDir) {
+      logger.error('The Forest configDir option you configured does not seem to be an existing directory.');
+    }
```
Comment (arnaud-moncel) [inline src/index.js:291]: As we remove the return we should put a `else if` here ;) 
```suggestion
    if (fs.existsSync(directorySmartImplementation)) {
      await requireAllModels(directorySmartImplementation);
    } else if (opts.configDir) {
      logger.error('The Forest configDir option you configured does not seem to be an existing directory.');
    }
```

**Review by arnaud-moncel — CHANGES_REQUESTED** [review]
Just found a little bug

**Review by arnaud-moncel — APPROVED** [review]
👌

**Pull request merged**